01-good-old-times/tf-01.py

```diff
@@ -35,7 +35,7 @@ def touchopen(filename, *args, **kwargs):
 data = [f.read(1024).split(',')] # data[0] holds the stop words
 f.close()
 
-data.append([])    # data[1] is line (max 80 characters)
+data.append(None)    # data[1] is line (max 80 characters)
```

Comment:
Two spaces too much before comment

Reply:
It's true that it doesn't need to be a list. However, a list (or better yet, an array, if Python had them) is more true to this style. This really is how you would access data in old times and in Assembly -- a few bytes at a time. Things like data[1][0] are to show the youngsters that you had to work hard to access data!